internal/statesync/syncer.go

```diff
@@ -169,7 +171,6 @@ func (s *syncer) SyncAny(discoveryTime time.Duration, retryHook func()) (sm.Stat
 			if discoveryTime == 0 {
 				return sm.State{}, nil, errNoSnapshots
 			}
-			retryHook()
```

Comment:
Why do we remove the retry hook here?

Reply:
Because snapshots have a pretty high priority - we shouldn't need to worry about having to retry. Or at least not after a couple of minutes

Reply:
what if a node restarts and misses the message?

Reply:
node as in a peer or the node doing the state sync itself?